oneview/resource_logical_interconnects.go

```diff
@@ -340,19 +403,21 @@ func resourceLogicalInterconnect() *schema.Resource {
 			"update_type": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
 			},
 		},
 	}
 }
 
 func resourceLogicalInterconnectRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
-	id := d.Get("name").(string)
```

Comment:
why have me removed getbyName for LI Resource?

Reply:
LI resource uses id(uuid) for getting the resources. Import works with id and not with name. Update of the resource also works by id and not by name